docs/dm/zh_CN/features/binlog-filter.md

```diff
@@ -123,4 +124,22 @@ filters:
     action: Ignore
 ```
 
-- `filter-procedure-rule` 过滤所有匹配到 pattern `test_*`.`t_*` 的 table 的 `^CREATE\\s+PROCEDURE`、`^DROP\\s+PROCEDURE` 操作
\ No newline at end of file
+- `filter-procedure-rule` 过滤所有匹配到 pattern `test_*`.`t_*` 的 table 的 `^CREATE\\s+PROCEDURE`、`^DROP\\s+PROCEDURE` 操作
+
+---
+
+#### 过滤 TiDB parser 不支持的 SQL 语句
+
+对于 TiDB parser 不支持的 SQL 语句，DM 无法解析获得 `schema`/`table` 信息，因此需要使用全局过滤规则（`schema-pattern: "*"`）。
+
+注意：全局过滤规则必须设置尽可能严格以避免不预期地过滤需要同步的数据
+
+设置下面的规则过滤 TiDB parser 不支持的 `PARTITION statement`
+
+```yaml
+filters:
+  filter-partition-rule:
+    schema-pattern: "*"
+    sql-pattern: ["ALTER\\s+TABLE[\\s\\S]*ADD\\s+PARTITION", "ALTER\\s+TABLE[\\s\\S]*DROP\\s+PARTITION"]
```

Comment:
If the YAML parser works correctly, you can avoid these double backslashes by using [single quote](https://yaml.org/spec/1.2/spec.html#id2776092).

```suggestion
    sql-pattern: ['ALTER\s+TABLE[\s\S]*ADD\s+PARTITION', 'ALTER\s+TABLE[\s\S]*DROP\s+PARTITION']
```

Reply:
:+1: yeah, I tested it and our YAML parser (*gopkg.in/yaml.v2*) works correctly.
but our users maybe not familiar with YAML, and can not differ single quote and double quote well, so I still want to keep them uniform.